data/events/2022-eindhoven.yml

```
@@ -0,0 +1,114 @@
+name: "2022-eindhoven" # The name of the event. Four digit year with the city name in lower-case, with no spaces.
+year: "2022" # The year of the event. Make sure it is in quotes.
+city: "Eindhoven" # The displayed city name of the event. Capitalize it.
+event_twitter: "devopsdaysehv" # Change this to the twitter handle for your event such as devopsdayschi or devopsdaysmsp
+description: "Devopsdays is coming to Eindhoven!" # Edit this to suit your preferences
+ga_tracking_id: "" # If you have your own Google Analytics tracking ID, enter it here. Example: "UA-74738648-1"
+
+# All dates are in unquoted 2022-MM-DDTHH:MM:SS+TZ:TZ, like this:
+#   variable: 2019-01-04T00:00:00+02:00
+#   variable: 2019-01-05T23:59:59+02:00
+# Note: we allow 2022-MM-DD for backward compatibility, but it can lead to unexpected behaviors (like your event disappearing from the front page during your last day)
+
+startdate: 2022-10-19T00:00:00+02:00 # The start date of your event. Leave blank if you don't have a venue reserved yet.
+enddate: 2022-10-21T00:00:00+02:00 # The end date of your event. Leave blank if you don't have a venue reserved yet.
+
+# Leave CFP dates blank if you don't know yet, or set all three at once.
+cfp_date_start: 2021-07-07T00:00:00+02:00 # start accepting talk proposals.
+cfp_date_end: 2021-09-21T00:00:00+02:00 # close your call for proposals.
```

These dates are in the past. I _think_ you meant 2022…
```suggestion
cfp_date_start: 2022-07-07T00:00:00+02:00 # start accepting talk proposals.
cfp_date_end: 2022-09-21T00:00:00+02:00 # close your call for proposals.
```

---

The CFP has been open since 2021, but you are right.. need to update the close to 2022.